rep-2000.rst

```diff
@@ -1,20 +1,69 @@
 REP: 2000
-Title: ROS 2 Target Platforms
-Author: Mikael Arguedas <mikael@openrobotics.org>, Steven! Ragnarok <steven@openrobotics.org>
+Title: ROS 2 Releases and Target Platforms
+Author: Mikael Arguedas <mikael@openrobotics.org>, Steven! Ragnarok <steven@openrobotics.org>, Dirk Thomas <dthomas@openrobotics.org>
 Status: Active
 Type: Informational
 Content-Type: text/x-rst
-Created: 10-Apr-2018, 24-Apr-2018, 21-May-2018, 10-Dec-2018, 20-May-2019, 21-May-2019, 29-Aug-2019
+Created: 10-Apr-2018, 24-Apr-2018, 21-May-2018, 10-Dec-2018, 20-May-2019, 21-May-2019, 29-Aug-2019, 25-Feb-2020
 
 
 Abstract
 ========
 
-This REP defines target platforms for each ROS 2 Distribution Release.
+This REP defines timeline for future ROS 2 releases as well as the targeted platforms for each specific one.
 We define platforms to include both operating system releases (e.g. Ubuntu Xenial (16.04 LTS)) as well as major language releases (e.g. Python 3.5).
 The target platforms represent the set on which all core stacks are expected to work.
 Exceptions can be made for stacks that are intentionally platform-specific.
 
+Release Schedule
+================
+
+Note: the following applies to ROS 2 releases *after* Foxy Fitzroy.
+Before, releases were made more frequently but with shorter support due to the fact that many foundational parts of ROS 2 were still heavily iterated on.
+
+TL;DR ROS 2 release schedule post Foxy:
+* May 2021: G Turtle: non-LTS release, supported for 1.5 years
+* May 2022: H Turtle: LTS release, supported for 5 years
+* May 2023: I Turtle: non-LTS release, supported for 1.5 years
+* May 2024: J Turtle: LTS release, supported for 5 years
+* and so on, alternating annually between LTS and non-LTS releases
+
+Frequency
+---------
+
+New ROS 2 releases are published in a time based fashion **every 12 months**.
+The rationale is that a shorter cycle (like 6 months) results in a significant overhead and many active releases at the same time (assuming they have the same support length).
+On the other hand a longer cycle (like 2 years) is too long for users to wait for new features to be available in a ROS 2 release.
+
+Targeted Platforms
+------------------
+
+Since regular Ubuntu releases are only supported for 9 months ROS 2 is not targting those.
+A single ROS 2 distribution only **targets one Ubuntu LTS**.
```

Comment:
Can this be made clear about whether it is specifically the server or desktop release, or if the very minimal Ubuntu is also supported?

Reply:
All our test and release infrastructure uses Ubuntu Desktop. At the end of the day if you don't want to install any packages which have GUI it should be fine to use them on Ubuntu Server.

Regarding minimal: this basically boils down to the default set of packages which is installed. As soon as you install ROS packages they will depend on their necessary dependencies. So once you installed e.g. the ROS desktop metapackage you won't have a "minimal" Ubuntu installation anymore.

Therefore I don't think this needs to be mentioned in this document.